1 deleted test

mumbler6 opened the pull request:
Test was deleted in this commit `https://github.com/apache/ignite-3/commit/dd6ffb555b99d23b1484083e50831f934ee20405`

darko-marinov commented on pr-data.csv:
```diff
@@ -2383,7 +2383,7 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/page-memory,org.apache.ignite.internal.pagememory.freelist.AbstractFreeListTest.testMultiThread,ID,MovedToGradle,,https://github.com/apache/ignite-3/commit/c91f39cd80dfd143e5ac848486366ad6ed124ce5
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgs,ID,,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgsNullConfig,ID,,,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgsNullConfig,ID,Deleted,,https://github.com/apache/ignite-3/commit/dd6ffb555b99d23b1484083e50831f934ee20405
```
Skimming over https://github.com/apache/ignite-3/commit/dd6ffb555b99d23b1484083e50831f934ee20405 shows no changes in `IgniteCliRunner...`. What do you get with `git show --name-status dd6ffb555b99d23b1484083e50831f934ee20405 | grep IgniteCliRunner`.

mumbler6 replied:
Sorry I forgot to mention this, they changed the name of the test file to IgniteRunnerTest.java in some commits before. I thought it was more appropriate to list the delete in this commit where you'll see the `smokeTestArgsNullConfig` is deleted.

darko-marinov replied:
Let's list both, first some `MovedOrRenamed` for the test class and then later `Deleted` for the test method.

mumbler6 replied:
Do I make another entry in the csv with just the test class? With empty test method?

darko-marinov replied:
If the test class had the reported test method when the test class was moved, then keep the same class name and method name, just add `MovedOrRenamed` in the status (and the URL to the commit in `Notes`).

darko-marinov commented:
You got `This branch has conflicts that must be resolved`. Also, it seem you claim the same test (same class and method name) was both `MovedOrRenamed` and `Deleted` -- both can't happen. Maybe the test was first moved/renamed to a different name, and then that different name was deleted.

mumbler6 commented:
Ok I understand what you mean. I've changed the deleted test to a different path, `IgniteRunnerTest.java`. I also just noticed another test I marked as movetogradle was also movedorrenamed, so I changed the status to that one instead.

darko-marinov commented:
The branch cannot be rebased safely. Can you squash all your commits into one and rebase on the latest `main`.
